extensions/wikia/EditPageLayout/js/editor/AceEditor.js

```diff
@@ -176,6 +177,19 @@ define(
 		$('.editpage-widemode-trigger').click(editorModeChange);
 	}
 
+	function launchEditorPreview(){
+		var editorValue = ace.getContent(),
+			wikiaDomain = mw.config.get('wgServer').split('://')[1],
+			templateName = mw.Title(mw.config.get('wgPageName'), 'Template').title,
+			infoboxPreviewURL = 'http://localhost:8080/', // TODO: add to config
+			$form = $('<form>').attr({'action': infoboxPreviewURL, 'method': 'POST'});
```

Comment:
```
$('<form>')
.attr({'action': infoboxPreviewURL, 'method': 'POST'})
.append(
    $('<textarea>').val(editorValue).attr({'name': 'editor_value'}),
    $('<input>').val(wikiaDomain).attr({'name': 'wikia_domain'}),
    $('<input>').val(templateName).attr({'name': 'template_name'})
)
.submit()
```
Not saying that's a better way of doing it - but just showing it could be done this way - jQuery is awesome :)